src/main/java/com/flansmod/common/network/PacketReload.java

```diff
@@ -196,17 +196,7 @@ public void handleClientSide(EntityPlayer clientPlayer) {
                     }
                 }
             }
-            //If there was a valid non-empty magazine / bullet item somewhere in the inventory, take one to put in the gun
```

Comment:
For anyone else seeing this:

The theory is this is already performed on the server, in reload() called by the server. Doing it on the client just introduces a discrepancy that gets fixed on an inventory changed.

Removing it entirely seems to fix it, but I don't know enough about inventory sync stuff to know if this has any other consequences